controller/customdomain_controller.go

```diff
@@ -277,15 +277,27 @@ func (r *CustomDomainReconciler) Reconcile(ctx context.Context, request ctrl.Req
 				return reconcile.Result{}, err
 			}
 			isAWS := *cloudPlatform == "AWS"
+			var lbType operatorv1.AWSLoadBalancerType
```

Comment:
This is an empty var, so line 283 will never be hit. You need to use the value from the `customIngress.Spec`